app/create_file.py

```diff
@@ -1 +1,56 @@
-# write your code here
+from datetime import datetime
+from sys import argv
+from os import mkdir, chdir
+
+filename = ""
+directories = []
+
+if "-f" in argv:
+    index = 0
+
+    for _ in range(len(argv) - 1):
+        index += 1
+        if argv[_: _ + 1] == ["-f"]:
```

Comment:
list slice is unnecessary here. Following will do the same while it's easier to read. Also array wont be created at every step of a loop.
```suggestion
        if argv[ _ ] == "-f":
```